prompt2model_demo.py

```diff
@@ -181,8 +181,48 @@ def main():
             TaskType.TEXT_GENERATION, status["instruction"], status["examples"]
         )
         line_print("Retrieving dataset...")
+        line_print("Do you want to perform data transformation? (y/n)")
+        line_print(
+            "Data transformation converts retrieved data into desired format as per prompt"  # noqa E501
```

Comment:
Grammar/punctuation suggestion
```suggestion
            "Data transformation converts retrieved data into the desired format as per the prompt."  # noqa E501
```